components/automate-gateway/handler/applications.go

```diff
@@ -24,18 +26,24 @@ func NewApplicationsHandler(applicationsClient applications.ApplicationsServiceC
 // GetServiceGroupsHealthCounts returns the health counts from all service groups
 func (a *Applications) GetServiceGroupsHealthCounts(
 	ctx context.Context,
-	in *applications.ServiceGroupsHealthCountsReq) (*applications.HealthCounts, error) {
+	request *applications.ServiceGroupsHealthCountsReq) (*applications.HealthCounts, error) {
 
-	inDomain := &applications.ServiceGroupsHealthCountsReq{}
-	out := &applications.HealthCounts{}
-	f := func() (proto.Message, error) {
-		return a.client.GetServiceGroupsHealthCounts(ctx, inDomain)
-	}
-	err := protobuf.CallDomainService(in, inDomain, f, out)
+	log.WithFields(log.Fields{
```

Comment:
We shouldn't need this in the gateway: any GRPC call will be logged if
- its status is not "OK", or
- we're in debug mode

[see here for the logging middleware doing that](https://github.com/chef/automate/blob/a9aa5bf7f586090a532b71947e546446c64182e4/components/automate-gateway/gateway/server.go#L267-L287)